im.bernard.Nostalgia.json

```
@@ -10,18 +10,87 @@
         "--socket=wayland"
     ],
     "modules" : [
+        {
+            "name": "libsass",
+            "buildsystem": "meson",
+            "sources": [{
+                "type": "git",
+                "url": "https://github.com/lazka/libsass.git",
+                "commit": "302397c0c8ae2d7ab02f45ea461c2c3d768f248e"
+            }]
+        },
+        {
+            "name": "sassc",
+            "buildsystem": "meson",
+            "sources": [{
+                "type": "git",
+                "url": "https://github.com/lazka/sassc.git",
+                "commit": "82803377c33247265d779af034eceb5949e78354"
+            }]
+        },
+        {
+            "name" : "pango",
+            "buildsystem": "meson",
+            "config-opts" : [
+                "-Dintrospection=disabled"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://gitlab.gnome.org/GNOME/pango/-/archive/1.50.3/pango-1.50.3.tar.gz",
+                    "sha256" : "4a8b0cf33d5f9ecaa9cd99dd72703d5c4c53bc58df64dd9538493bb4356ab691"
+                }
+            ]
+        },
+        {
+            "name": "gtk",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dgtk_doc=false",
+                "-Dbuild-tests=false",
+                "-Dbuild-examples=false"
+            ],
+            "sources": [{
+                "type": "git",
+                "url": "https://gitlab.gnome.org/GNOME/gtk.git",
+                "tag": "4.6.0",
+                "commit": "70cb61fb7104c76a15bc6494a10e6ff1d470f6d8"
+            }]
+        },
+        {
+            "name": "libadwaita",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dintrospection=disabled",
+                "-Dgtk_doc=false",
+                "-Dtests=false",
+                "-Dexamples=false",
+                "-Dvapi=false"
+            ],
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig"
+            ],
+            "sources": [{
+                "type": "git",
+                "url": "https://gitlab.gnome.org/GNOME/libadwaita.git",
+                "tag": "1.0.1",
+                "commit": "b274a0c32c51a92692e3b282f04e88f033ba73b1"
+            }]
+        },
         {
             "name" : "libportal",
             "buildsystem" : "meson",
             "config-opts" : [
-                "-Dgtk_doc=false"
+                "-Dgtk_doc=false",
```

```suggestion
                "-Ddocs=false",
```

It changed in 0.5, that's why the builld is failing